app/containers/ItunesContainer/tests/index.test.js

```diff
@@ -58,6 +60,23 @@ describe('<ItunesContainer /> container tests', () => {
     expect(clearItunesDataSpy).toBeCalled();
   });
 
+  it('should call dispatchItunesData on onSearch submit', async () => {
+    const ituneName = 'test';
+    const { getByTestId } = renderProvider(<ItunesContainer dispatchItunesData={submitSpy} />);
+    fireEvent.keyDown(getByTestId('search-bar'), { keyCode: 13, target: { value: ituneName } });
+
+    await timeout(500);
+    expect(submitSpy).toBeCalledWith(ituneName);
+  });
+
+  it('should  dispatchItunesData on update on mount if ituneName is already persisted', async () => {
```

Comment:
please correct the typo in the test name

Reply:
updated